src/bedriftsmeny/byggOrganisasjonsTre.ts

```diff
@@ -0,0 +1,94 @@
+import {
+    JuridiskEnhetMedUnderEnheterArray,
+    ListeMedJuridiskeEnheter,
+    Organisasjon,
+    tomAltinnOrganisasjon
+} from "./Organisasjon";
+
+export async function byggOrganisasjonstre(
+    organisasjoner: Organisasjon[]
+): Promise<JuridiskEnhetMedUnderEnheterArray[]> {
```

Comment:
Denne funksjonen er lissom kjernen i hele PR-en. Metoden er komplisert, og det må den nesten være. Jeg synes den er skrevet bra; når jeg leser gjennom metoden forstår jeg hva som skjer, og hvordan man kommer fram til returverdien. Men det er likevel noen grep vi kan ta for å øke lesbarheten enda mer, og rett og slett bare gjøre det kortere å lese gjennom.
1. Være konsekvent med navngivning. Enten bruke `orgnr` eller `orgNr`, enten`juridiskEnhet` eller `jurEnhet` osv. Da slipper jeg å tenke så mye 😛 
2. Bruke inline lambda-funksjoner. Personlig synes jeg det er vanskeligere å lese 
```
const underenheter = organisasjoner.filter(function(organisasjon: Organisasjon) {
        return organisasjon.OrganizationForm === 'BEDR' && organisasjon.OrganizationNumber;
    });
```
enn 
```
const underenheter = organisasjoner.filter(organisasjon =>  organisasjon.OrganizationForm === 'BEDR' && organisasjon.OrganizationNumber);
```
3. Lage flere hjelpefunksjoner. F.eks. vet jeg fra før at `orgtre.sort((a, b) => a.JuridiskEnhet.Name.localeCompare(b.JuridiskEnhet.Name));` betyr "sortér alfabetisk", men jeg måtte tenke meg om to ganger, og for noen som ikke har gjort det før kan det fort ta flere minutter å finne ut hva det betyr. Funksjonen `finnJuridiskeEnheter` kan vel også egentlig flyttes ut av `byggOrganisasjonstre`? Så slipper man å forholde seg til ekstra logikk 🙂